packages/solid-crs-pods/assets/register.html.ejs

```diff
@@ -0,0 +1,11 @@
+<h1>Registreren</h1>
+<form method="post" id="mainForm">
+
+  <% if (locals.message) { %>
+    <p class="error">Error: <%= message %></p>
+  <% } %>
+
+  <%- include('./register-partial.html.ejs', { allowRoot: false }) %>
+
+  <p class="actions"><button type="submit" name="submit">REGISTREER</button></p>
```

Comment:
@lem-onade I remember a comment in Figma saying the button should be in uppercase but looking at the other screens all the buttons seem to be lowercase with a capital letter?
```suggestion
  <p class="actions"><button type="submit" name="submit">Registreer</button></p>
```
```suggestion
  <p class="actions"><button type="submit" name="submit">REGISTREER</button></p>
```